### PR TITLE
Updated OAuth2 provider and organization fields in account creation e-mails

### PR DESCRIPTION
**Pull request by emmdurin**
Sending User Details including provider name and provider Uid and organization to console when creating a new OAuth2 account.

### CODE REVIEW

File: gateway/src/main/java/org/georchestra/gateway/accounts/events/rabbitmq/RabbitmqAccountCreatedEventSender.java [hunk 1]
```diff
@@ -48,19 +48,22 @@ public void on(AccountCreated event) {
         if (null != oAuth2ProviderId) {
             String fullName = user.getFirstName() + " " + user.getLastName();
             String email = user.getEmail();
-            String provider = oAuth2ProviderId;
-            sendNewOAuthAccountMessage(fullName, email, provider);
+            String organization = user.getOrganization();
+            String[] providerFields = oAuth2ProviderId.split(";");
+            sendNewOAuthAccountMessage(fullName, email, organization, providerFields[0], providerFields[1]);
```
Comment (pmauduit) [inline gateway/src/main/java/org/georchestra/gateway/accounts/events/rabbitmq/RabbitmqAccountCreatedEventSender.java:53]: no OOB checks ? (`ArrayIndexOutOfBoundsException`)

**Review by pmauduit — COMMENTED** [review]
I did not get what was the intent behind the PR, I assume it is to add some extra information to the message sent via rabbitMQ, so that the console can use these infos when sending a notification when an account is created ?

About the oauth2providerId, is it a convention to have a string with a semicolon, so that we can be sure we will be able to split the string on this character ?

File: gateway/src/main/java/org/georchestra/gateway/accounts/events/rabbitmq/RabbitmqAccountCreatedEventSender.java (hunk 1 above)
Comment (emmdurin) [reply gateway/src/main/java/org/georchestra/gateway/accounts/events/rabbitmq/RabbitmqAccountCreatedEventSender.java:53]: You're right about the goal of this change, and yes, the semicolon is a convention in the gateway for this field. I can add OOB check, I prefer with a if on length.

**Review by pmauduit — APPROVED** [review]

**Comment by pmauduit** [comment]
As discussed this morning, it may deserve 2 different fields instead of splitting the oauth2ProviderId variable

**Review by pmauduit — APPROVED** [review]

**Comment by pmauduit** [comment]
> As discussed this morning, it may deserve 2 different fields instead of splitting the oauth2ProviderId variable

This can be revisited later on, LGTM then

**Pull request merged**